wasmer/test/testdata/log_import_memory.wat

```diff
@@ -0,0 +1,15 @@
+(module
+  (type (;0;) (func (param i32 i32)))
+  (type (;1;) (func))
+  (import "env" "log_message" (func (;0;) (type 0)))
+  (import "env" "memory" (memory (;0;) 17))
+  (func (;1;) (type 1)
+    i32.const 1048576
```

Comment:
We don't read from the memory here, but from the statically allocated data, i.e. from the Data Section.

Reply:
Ouch. Interesting it works... I'll fix it to read from memory.